src/infinite-list/index.tsx

```diff
@@ -67,13 +69,17 @@ const InfiniteList = (props: InfiniteListProps, ref: any) => {
   const reloadPagesDebounce = useCallback(debounce(reloadPages, 500, {leading: false, trailing: true}), [reloadPages]);
 
   useEffect(() => {
+    if (props.disableScrollOnDataChange) {
```

Comment:
instead of extracting using props.xxx you can add it above where we extract all props. Also for the prop you're passing to the RecyclerListView